src/module/chat-message/helpers.ts

```diff
@@ -190,4 +190,21 @@ function toggleOffShieldBlock(messageId: string): void {
     CONFIG.PF2E.chatDamageButtonShieldToggle = false;
 }
 
-export { isCheckContextFlag, applyDamageFromMessage };
+/**
+ * Show or hide a clear-measured-template button on a message (applicable to spell cards with template-placed buttons).
+ * The button will be shown if templates are placed and the user has ownership; otherwise it will be hidden.
+ */
+function toggleClearTemplatesButton(message: ChatMessagePF2e | null): void {
+    if (!message || !canvas.ready) return;
+
+    const selector = `li[data-message-id="${message.id}"] button[data-action=spell-template-clear]`;
+    for (const chatLogDOM of htmlQueryAll(document.body, "#chat-log, #chat-popout")) {
+        const clearTemplatesButton = htmlQuery(chatLogDOM, selector);
+        if (!clearTemplatesButton) continue;
+        const hasMeasuredTemplates = !!canvas.scene?.templates.some((t) => t.message === message && t.isOwner);
+        const method = hasMeasuredTemplates ? "remove" : "add";
+        clearTemplatesButton.classList[method]("hidden");
```

Comment:
```suggestion
        clearTemplatesButton.classList.toggle("hidden", !hasMeasuredTemplate);
```

Though there is also the `hidden` attribute you could set which is part of the web start, but unfortunately it loses in priority to all `display` css properties so you need to style it specifically if you go that route most of the time.

Reply:
The `hidden` class is from core (just has `display: none`).